publish/working-with-pipelines/working-with-pipelines.md

```diff
@@ -210,7 +210,21 @@ status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ```
 
-- Once you have the external route and the `ca.crt` for the internal image registry external route, use the steps in section, `Private image registries`.  @aadeshpa how do you get the ca.crt?  can we add a pointer here?
+- Once you have the external route setup you can find the ca.crt value for the external route as below from the cluster
+
+```
+oc get configmap  -n openshift-image-registry
+
+NAME                          DATA   AGE
+image-registry-certificates   2      7d
+serviceca                     1      7d
+trusted-ca                    1      7d
+[root@rugger-inf ~]# 
+
+```
+  From the above command results try to search the content of configmap `image-registry-certificates` and find relevant external route certificate.
```

Comment:
Change to:
Search the contents of the `image-registry-certificates` configmap to find the relevant external route certificate. Note the certificate value for the external route.

Reply:
looks good.